prime-router/src/test/resources/metadata/tables/npi-lookup.csv

```diff
@@ -0,0 +1,9 @@
+first_name,last_name,ordering_provider_id,testing_lab_clia,sender_id,default
+Paul,Fineburg,1023040318,01D2079572,cuc-al,true
+Richard,Oyler,1013012657,01D2029146,cuc-al,true
+Stanley,Clayton,1174049449,,cuc-al,false
+Crystal,Morgan,1295219632,,cuc-al,false
+Mark,Minton,1376788521,,cuc-al,false
+Nikki,Quinley,1467610550,,cuc-al,false
+Carl,Parks,1538116132,,cuc-al,false
+Jonathan,Parker,1568710499,,cuc-al,false
```

Comment:
this looks great! thanks for putting this together :slightly_smiling_face:

Reply:
No prob!